src/oph/heratepalvelu/tep/jaksoHandler.clj

```diff
@@ -213,14 +207,15 @@
                :kasittelytila         [:s (:ei-niputettu c/kasittelytilat)]
                :sms_kasittelytila     [:s (:ei-lahetetty c/kasittelytilat)]
                :niputuspvm            [:s (str niputuspvm)]}]
-          (if (check-open-keskeytymisajanjakso herate)
+          (if (and (:keskeytymisajanjaksot herate)
```

Comment:
Jos tää lisächeck oikeasti tarvitaan, pitäisi logiikka ehkä kääntää taas toisin päin ja nimetä predikaatti esim. `has-open-keskeytymisajanjakso?`

Reply:
tähän nyt palattiin kumminkin täällä #237 :)